--- pipewire.conf	2024-07-12 02:21:06.289359997 +0300
+++ pipewire.conf.new	2024-07-12 02:31:54.424506636 +0300
@@ -41,7 +41,7 @@
 
     ## Properties for the DSP configuration.
     #default.clock.rate          = 48000
-    #default.clock.allowed-rates = [ 48000 ]
+    default.clock.allowed-rates = [ 41000, 48000, 88200, 96000, 192000, 176400, 352800, 384000 ]
     #default.clock.quantum       = 1024
     #default.clock.min-quantum   = 32
     #default.clock.max-quantum   = 2048
